language-server/modules/langserver-core/src/test/resources/completion/query_expression/source/query_expr_ctx_source19.bal

```diff
@@ -0,0 +1,15 @@
+function testIterableOperation() {
+    Customer c1 = {id: 1, name: "Melina", noOfItems: 12};
+    Customer c2 = {id: 2, name: "James", noOfItems: 5};
+    Customer c3 = {id: 3, name: "Anne", noOfItems: 20};
+
+    Customer[] customerList = [c1, c2, c3];
+
+    Customer[] filtered = from var c in customerList w
```

Comment:
To be on the safe side in the future, let's create a new test having the following in the next line,
```
module1:TestRecord1 rec = {
    recField1: 10
}
```
You need the import `import ballerina/module1`. I just want to make sure we cover following cases,
1. We consider having a query pipeline in the last statement of a function body
2. Having a query pipeline in an intermediate statement of a function body

That way, we will most probably notice when the parser behavior changes (As per my understanding, a parser rewrite is planned as well).

Reply:
Done!